components/SVG/FruitBowl/FruitBowl.tsx

```diff
@@ -0,0 +1,45 @@
+type FruitBowlProps = {
+    width?: number;
+    height?: number;
+    className?: string;
+};
+
+export const FruitBowl = ({
+    width = 308,
+    height = 285,
+    className,
+}: FruitBowlProps) => (
+    <svg
+        xmlns="http://www.w3.org/2000/svg"
+        xmlnsXlink="http://www.w3.org/1999/xlink"
+        className={className}
+        width={width}
+        height={height}
+        viewBox="0 0 308 285"
+        fill="none"
+    >
+        <path
+            fill="url(#a)"
```

Comment:
**Question**: I'm not sure how this is being used?

Reply:
This is awkward SVG setup / exporting from figma - this path is rotating an image of the fruit bowl file, I'm currently waiting on getting a new file for this SVG. I will add a note to a ticket to fix this up later.